CIP-1694/README.md

```diff
@@ -1109,16 +1107,19 @@ Firstly, during the bootstrap phase, a vote from the constitutional committee
 is sufficient to change the protocol parameters.
 Secondly, during the bootstrap phase, a vote from the constitutional committee,
 together with a sufficient SPO vote, is sufficient to initiate a hard fork.
-No other actions are possible during the bootstrap phase.
-
-The bootstrap phase ends when a given number of epochs has elapsed,
-as specified in the next ledger era configuration file.
-This is likely to be a number of months after the hard fork.
-
-Moreover, there will be an interim Constitutional committee,
-also specified in the next ledger era configuration file,
-whose term limits will be set to expire when the bootstrap phase ends.
-The rotational schedule of the first non-bootstrap committee could be included in the constitution itself.
+Info actions will be available, but
+no other actions are possible during the bootstrap phase.
```

Comment:
"Info actions will be available, but no other actions are possible during the bootstrap phase."

maybe this is confusing phrasing, as both hardforks and protocol parameter change governance actions are also supported.

Reply:
resolving: fixed in https://github.com/cardano-foundation/CIPs/pull/800/commits/9ab965eea626d4fd091e898d6f4e21aabdd02c63